python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ip_tacacs_source_interfaces.schema.yml

```
@@ -9,6 +9,8 @@ keys:
   ip_tacacs_source_interfaces:
     type: list
     display_name: IP Tacacs Source Interfaces
+    unique_keys:
+      - vrf
```

Discussed with @ClausHolbechArista - given that it does not make sense to have the same source interface in multiple VRFs (even if this is allowed by EOS...) we were discussing even adding this:

```suggestion
    primary_key: name
    unique_keys:
      - vrf
```

---

ok so change of plan looking at the CLI further. We keep it as you have but add a comment for 6.0 to make VRF a primary_key (and the templates will need to be adjusted)